selection_sort.go

```diff
@@ -0,0 +1,16 @@
+package sort
+
+func selectionSort(arr []int) []int {
```

Comment:
The function `selectionSort` should be exported (i.e., start with a capital letter) if it is intended to be used outside of the current package. Consider renaming it to `SelectionSort` for better visibility and consistency with Go naming conventions.